Dockerfile

```diff
@@ -1,10 +1,6 @@
 FROM node:6-onbuild
 MAINTAINER Trent Oswald (@therebelrobot) <trentoswald@therebelrobot.com>
-RUN npm run install:clean --production
-
-EXPOSE 5465
-
-ENV MONGODB mongodb://localhost:27017/brigadehub-docker
+RUN npm install
 
 ENV GITHUB_ID be1b409d62f41a56684c
```

Comment:
@therebelrobot I didn't mean to offend. If you look at the issue related to this, I mention that we should look into this more, but in a different issue/commit. I just wanted to call it out here (which probably isn't the right place either). 

I am not disagreeing that it works or not. My concern is if I can take these keys and start deleting repos in sfbrigadehub's account (which I should have explained better, but was in a rush).